tests/test_git_init.py

```diff
@@ -2,49 +2,31 @@
 
 import pathlib
 import subprocess
+import typing
 
 import pytest
 
 
 @pytest.mark.parametrize(
-    "git_init_cookiecutter_option",
-    ["initialise_git_repository=True", "initialise_git_repository=False"],
+    "git_init",
+    [{"initialise_git_repository": True}, {"initialise_git_repository": False}],
```

Comment:
I think having two dictionaries for a single parametrized argument is more complicated/messier than just doing a simple parametrize like I was doing before. In addition to this way being more complex, `pytest` won't generate nice test names. Before it would have been `test_initialisation_of_git_repo[git_init=True]`, but now it would be `test_initialisation_of_git_repopgit_init[{"initialise_git_repository": True}]`.

Reply:
🤔 That is true. 

And the output of the tests in the logs will be read more often than the code. So I now tend to agree. 

It's really just one `git revert` able commit. So I'm fine with going back.

Reply:
I let Matt and Paddy express an opinion.

Reply:
Does ☝️ mean you are happy with reverting this change, but then "True"/"False" should still be changed to bools instead of strings?

Reply:
Yes

Reply:
(as long as it works, which I believe it should)

Reply:
It does... [fde687b](https://github.com/UCL-ARC/python-tooling/pull/405/commits/fde687b7fb127d1bf58cae33f1d0ca21246ec6d6).